update-license/main.go

```diff
@@ -80,19 +72,61 @@ func main() {
 
 func do() error {
 	flag.Parse()
+
 	if len(flag.Args()) < 1 {
 		return fmt.Errorf("usage: %s GO_FILES", os.Args[0])
 	}
 
-	return updateFiles(flag.Args(), time.Now().UTC().Year(), *flagDryRun)
+	return updateFiles(
+		flag.Args(),
+		time.Now().UTC().Year(),
+		*flagLicense,
+		*flagOwner,
+		*flagDryRun,
+	)
 }
 
-func updateFiles(filePaths []string, year int, dryRun bool) error {
+func fullLicense(ts string, year int, owner string) string {
+	var buf bytes.Buffer
+	t, err := template.New("").Parse(ts)
+	if err != nil {
+		log.Fatal("failed to parse license remplate", err)
```

Comment:
Generally, for unrecoverable issues, I prefer panic instead of log.Fatal so
that we still get `defer` cleanup. `os.Exit` outside `main()` is annoying to
debug/track down.

(Also, typo)

```suggestion
		log.Panic("failed to parse license template", err)
```